specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerConnections.json

```diff
@@ -0,0 +1,304 @@
+              "$ref": "#/definitions/NetworkManagerConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Network Manager Connection": {
+            "$ref": "./examples/NetworkManagerConnectionGet.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkManagerConnections"
+        ],
+        "operationId": "NetworkManagerConnections_Delete",
+        "description": "Delete a specified pending connection created by this subscription/management group.",
+        "responses": {
+          "200": {
+            "description": "Delete Succeed."
+          },
+          "204": {
+            "description": "Request successful. The resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Network Manager Connection": {
+            "$ref": "./examples/NetworkManagerConnectionDelete.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Network/networkManagerConnections": {
+      "get": {
+        "tags": [
+          "NetworkManagerConnections"
+        ],
+        "operationId": "NetworkManagerConnections_List",
+        "description": "List all network manager connections created by this subscription/management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ListTopParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ListSkipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the network manager connection(s).",
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Network Manager Connection": {
+            "$ref": "./examples/NetworkManagerConnectionList.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Network/listPendingScopeConnections": {
```

Comment:
>listPendingScopeConnections [](http://example.com/codeflow?start=42&length=27)

Is the intention here to enalbe the for use this POST action in an arm template ?

Reply:
This is strictly a discoverability API. It is used to list any scopeConnections which reference the base scope in the uri. As opposed to the standard GET list scope sconnections api, which will list all scope connections under the parent network manager resource.